lms/djangoapps/learner_recommendations/test_views.py

```diff
@@ -0,0 +1,134 @@
+"""
+Tests for Learner Recommendations views and related functions.
+"""
+
+import json
+from unittest import mock
+
+from django.urls import reverse_lazy
+from rest_framework.test import APITestCase
+
+from common.djangoapps.student.tests.factories import UserFactory
+
+
+class TestAlgoliaCoursesSearchView(APITestCase):
```

Comment:
lets use the "@skip_unless_lms" decorator

Reply:
I think it is handled now in: https://github.com/openedx/edx-platform/blob/master/.github/workflows/unit-test-shards.json. It is being picked up by one of the lms shards now.